CMSgov/managed-care-review
# Add last updated to CMS Dashboard

## Summary
- remove CMS_DASHBOARD feature flag
- Add last updated column and in general move from looking at current revision to previous submitted revision for form data. That's something I overlooked in the first pass at the dashboard, likely this would cause bugs.

#### Related issues
https://qmacbis.atlassian.net/browse/MR-2466

#### Test cases covered
**Jest**
see `CMSDashboard.test.tsx` (web tests)
 - displays no submission text when no submitted packages exist
- displays submissions table when submitted packages exist
- displays submissions table with expected headers
- displays submissions table excluding any in progress drafts
- displays submission type as expected for current revision that is submitted/resubmitted
- displays the expected program tags for current revision that is submitted/resubmitted
- displays each health plan package status tag as expected
- displays name, type, programs and last update based on previously submitted revision for UNLOCKED package

**Cypress**
- CMS user can view submission in the CMS dashboard

## QA guidance
- Please test this work thoroughly with unlock and resubmit to make sure that state draft edits  data is not appearing in the dashboard. This is most relevant when an unlocked package is in progress of being edited
- check that last updated column displays as expected
